python/cucim/src/cucim/__init__.py

```diff
@@ -30,20 +30,49 @@
     Functions from scikit-image.
 
 """
+_is_cupy_available = False
+_is_clara_available = False
 
 # Try to import cupy first.
 # If cucim.clara package is imported first, you may see the following error when running on CUDA 10.x (#44)
 #   python3: Relink `/usr/lib/x86_64-linux-gnu/libnccl.so.2.8.3' with `/lib/x86_64-linux-gnu/librt.so.1' for IFUNC symbol `clock_gettime'
 #   Segmentation fault
 try:
     import cupy
+    _is_cupy_available = True
 except ImportError:
     pass
 
 try:
     from .clara import CuImage, __version__, cli
+    _is_clara_available = True
 except ImportError:
     from ._version import get_versions
     __version__ = get_versions()['version']
     del get_versions
     del _version
+
+
+def is_available(module_name: str = "") -> bool:
```

Comment:
Should we add a test for this function?

Reply:
Thanks for the feedback!
Added unit tests in the latest patch.